### DISCUSSION TITLE
Remote scheduler

### DISCUSSION THREAD

**Discussion by robertstefan** [quartznet/quartznet#506] [General] [answered]
Hello, is there an example of a RemoteSchedulure creation for v2.5.0?

What I am trying to achieve is to connect from a WinService running on a machine, to a Quartz.NET scheduler that is running on another machine, over the network.
At first I was trying to connect to the scheduler by using the default implementation 
```csharp
string serverAddress = $"tcp://{connectorOptions.Server}:{connectorOptions.Port}/{connectorOptions.SchedulerName}";
		
NameValueCollection properties = new NameValueCollection
{
	["quartz.scheduler.instanceName"] = "RemoteClient",
	["quartz.scheduler.proxy"] = "true",
    ["quartz.threadPool.threadCount"] = "0",
	["quartz.scheduler.proxy.address"] = serverAddress
};

ISchedulerFactory schedulerFactory = new StdSchedulerFactory(properties);

try
{
	_scheduler = schedulerFactory.GetScheduler();
}
catch (SchedulerException ex)
{
	Helpers.Logger.Error("Unable to connect to the specified server, ", ex);
}
```

but every time I wanted to register a job I ended up receiving an error that it was not possible. 😞 
Anybody that can point me in the right direction?

Thank you.

**Comment by lahma** [comment, marked as answer]
You should check how [example 12](https://github.com/quartznet/quartznet/tree/2.x/src/Quartz.Examples/example12) works. Make sure you have the same job dlls deployed on client and server side.
